app/scripts/modules/core/delivery/status/executionStatus.component.ts

```diff
@@ -0,0 +1,61 @@
+import {IComponentController, IComponentOptions, module} from 'angular';
+
+import {EXECUTION_USER_FILTER} from './executionUser.filter';
+import {IExecution} from 'core/domain/IExecution';
+
+
+import './executionStatus.less';
+
+export class ExecutionStatusController implements IComponentController {
+  public execution: IExecution;
+  public toggleDetails: () => void;
+  public showingDetails: () => boolean;
+  public standalone: boolean;
+
+  public filter: any;
+  public sortFilter: any;
+  public parameters: { key: string, value: any }[];
```

Comment:
Nice, this is way nicer (IMO) than declaring a new interface every time we need a simple typed object that's not used elsewhere.